packages/spotlight/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 The following is a curated list of changes in the Enact spotlight module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `ExpandableItem` and other expandable components to spotlight correctly when switching from
```

Comment:
Shouldn't this be in the `moonstone` changelog instead?

Reply:
:disappear: